packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -268,6 +268,7 @@ const DataGrid = forwardRef((props, ref) => {
     () => ({
       setFocusedRowCol,
       getFocusedCell() { return { row: focusedRow, col: focusedCol }; },
+      getGridRef() { return grid.current; },
```

Comment:
We could just use a refcallback pattern similar to what we have with the Terra button.  I don't think we need to leverage the imperative handle hook here if we truly need access to the table.